### PR TITLE
alerting: Improve documentation regarding linking and visualizing alert rules in dashboards

### PR DESCRIPTION
**Pull request by tonypowa**
**What is this feature?**

Commit 1: https://grafana.com/docs/grafana/next/alerting/fundamentals/#alert-rules Mention linking dashboards to Alerts, and add shortcode to [alerting demo dashboard in play](https://play.grafana.org/d/000000074/alerting?orgId=1)

Commit 2: https://grafana.com/docs/grafana/next/alerting/fundamentals/alert-rule-evaluation/state-and-health/#alert-rule-state Add link to alerting demo dashboard in play to showcase alert rule state with the alert list panel and a panel that contains an alert (heart icon in the dashboard)

Commit 3: https://grafana.com/docs/grafana/next/alerting/alerting-rules/ mention the config option to link to existing panel and add link to alerting demo dashboard in play

Commit 4: https://grafana.com/docs/grafana/next/alerting/alerting-rules/create-grafana-managed-rule/#add-annotations edit the link to panel bit, and add link to alerting demo dashboard in play

Commit 5: https://grafana.com/docs/grafana/next/alerting/alerting-rules/create-mimir-loki-managed-rule/#add-annotations edit the link to panel bit, and add link to alerting demo dashboard in play

**Why do we need this feature?**

Improve documentation regarding linking and visualizing alert rules in dashboards by adding content to the docs above and  showcase that feature by adding shortcode to link to play.grafana

### CODE REVIEW

**Comment by tonypowa** [comment]
@brendamuir 
Commit 1 pushed  --  will test it locally and update the ticket
Commits 2-5 will be added to this PR later

**Comment by tonypowa** [comment]
done ✅ 
preview of each commit

let me know what you think @brendamuir  @ppcano 

commit 1
image

commit 2
image

commit 3
image

commit 4
image

commit 5
image

File: docs/sources/alerting/fundamentals/_index.md [hunk 1]
```diff
@@ -41,6 +41,10 @@ Add labels to uniquely identify your alert rule and configure alert routing. Lab
 
 After alert rules are created, they go through various states and transitions.
 
+You can also link alert rules to panels, allowing you to visualize the alert conditions directly within your dashboards and gain insights into the metrics triggering the alerts.
```
Comment (ppcano) [inline docs/sources/alerting/fundamentals/_index.md:44]: I am for Not mentioning Play in this section.

File: docs/sources/alerting/fundamentals/alert-rule-evaluation/state-and-health.md [hunk 2]
```diff
@@ -54,6 +54,12 @@ An alert instance can be in either of the following states:
 | **NoData**   | No data has been received for the configured time window.                                     |
 | **Error**    | The error that occurred when attempting to evaluate an alert rule.                            |
 
+
+Grafana provides an Alert List panel that allows you to visualize the states of different alert instances in a centralized view. By adding this panel to your dashboards, you can monitor which alerts are firing, pending, or inactive, enabling a comprehensive overview of your alerting landscape and facilitating quicker response times to any issues.
```
Comment (ppcano) [inline docs/sources/alerting/fundamentals/alert-rule-evaluation/state-and-health.md:58]: I am for Not mentioning Play in this section.

File: docs/sources/alerting/alerting-rules/_index.md [hunk 3]
```diff
@@ -43,6 +43,10 @@ How do you want your alert to be evaluated?
 
 How do you want to route your alert? What kind of additional labels could you add to annotate your alert rules and ease searching?
 
+In this section you also have the option to link an alert to a visualization.
```
Comment (ppcano) [inline docs/sources/alerting/alerting-rules/_index.md:46]: I am for Not mentioning Play in this section.

**Review by ppcano — COMMENTED** [review]
I think The Play admonition could fit well after the instruction `Add a dashboard and panel link`, but not convinced about including it in the other sections.

**Review by brendamuir — APPROVED** [review]

**Pull request merged**